.github/workflows/main.yml

```diff
@@ -102,6 +102,9 @@ jobs:
       - name: registry login
         uses: ./.github/actions/registry-login-action
         with:
+          development-registry-endpoint: ${{ secrets.DEV_FROM_ACCOUNT_PUBLIC }}.dkr.ecr.us-east-1.amazonaws.com
```

Comment:
quick question: is it possible to have `.dkr.ecr.us-east-1.amazonaws.com` be a part of the secret so it doesn't have to be added here?

Reply:
@jp-dbt Originally we had it this way, but we were using a different key name (DEV_REGISTRY_ENDPOINT). Infra asked us to use the `DEV_FROM_ACCOUNT_PUBLIC` key instead, and they are the ones who set the values. There's precedent for this format elsewhere ([like here in cloud](https://github.com/dbt-labs/dbt-cloud/blob/master/.github/workflows/main.yml#L540)), so we may just want to update server to use this format all over instead of the registry endpoint.

Reply:
Ok gotcha!

Reply:
@dichenqiandbt see above-- we can go through and change them all to the env vars that do not include the suffix, but I'd prefer to do that in a separate PR to get this closed out.